sensor_msgs/Image rgb
sensor_msgs/Image deep
sensor_msgs/CameraInfo camera_info